sys/arm64/include/pte.h

```diff
@@ -91,7 +95,17 @@ typedef	uint64_t	pt_entry_t;		/* page table entry */
 #define	 ATTR_S2_MEMATTR_WT		0xa
 #define	 ATTR_S2_MEMATTR_WB		0xf
 
+#if __has_feature(capabilities)
+/*
+ * The aarch64 orr instruction only works when we split this into two
+ * values to be used in two orr instructions
+ */
+#define	ATTR_DEFAULT_LOW	(ATTR_AF | ATTR_SH(ATTR_SH_IS))
+#define	ATTR_DEFAULT_HIGH	(ATTR_LC_ENABLED | ATTR_SC)
+#define	ATTR_DEFAULT		(ATTR_DEFAULT_LOW | ATTR_DEFAULT_HIGH)
+#else
 #define	ATTR_DEFAULT	(ATTR_AF | ATTR_SH(ATTR_SH_IS))
+#endif
```

Comment:
```suggestion
/*
 * The aarch64 orr instruction only works when we split this into two
 * values to be used in two orr instructions
 */
#define	ATTR_DEFAULT_LOW	(ATTR_AF | ATTR_SH(ATTR_SH_IS))
#if __has_feature(capabilities)
#define	ATTR_DEFAULT_HIGH	(ATTR_LC_ENABLED | ATTR_SC)
#else
#define	ATTR_DEFAULT_HIGH	(0)
#endif
#define	ATTR_DEFAULT		(ATTR_DEFAULT_LOW | ATTR_DEFAULT_HIGH)
```

Reply:
I'm trying to reduce the diff to FreeBSD

Reply:
That's not a reason to write worse code, especially when you're already touching all the contexts where this matters.